app/src/main/java/com/github/polypoly/app/menu/JoinGameLobbyActivity.kt

```diff
@@ -32,28 +32,35 @@ import com.github.polypoly.app.game.GameLobby
 import com.github.polypoly.app.game.user.Skin
 import com.github.polypoly.app.game.user.Stats
 import com.github.polypoly.app.game.user.User
+import com.github.polypoly.app.global.GlobalInstances.Companion.remoteDB
 import com.github.polypoly.app.global.Settings.Companion.DB_GAME_LOBIES_PATH
-import com.github.polypoly.app.network.FakeRemoteStorage
 import com.github.polypoly.app.network.getAllValues
 import com.github.polypoly.app.network.getValue
-import com.github.polypoly.app.ui.theme.PolypolyTheme
 import kotlinx.coroutines.delay
 import timber.log.Timber
+import java.util.concurrent.CompletableFuture
 
 /**
  * Activity where the user can join a gameLobby
  */
-class JoinGameLobbyActivity : ComponentActivity() {
+class JoinGameLobbyActivity : MenuActivity("Join a game") {
+    companion object {
+        const val POLLING_INTERVAL = 5000L
```

Comment:
Are you sure you want to do polling instead of using a listener?

Reply:
this code came from #82. Actually, he wrote a TODO when polling happens, saying that we should at an end in fact use listeners :)!